src/enclave/Enclave/Enclave_private.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do we use this for anything? It doesn't seem to be referenced, anywhere, but if so, we'd better warn the user to change it.